src/main/scala/xebia/cakemix/AskTimeoutProvider.scala

```diff
@@ -17,7 +17,9 @@ trait AskTimeoutProvider {
 }
 
 object AskTimeoutProvider {
-  def fromConfig(config: Config, key: String): Timeout = {
+  private val AskTimeoutConfigKey = "xebia.cakemix.ask-timeout"
```

Comment:
For what it's worth, I agree with removing the config prefix. But for package names? Why not have it show that the guys at Xebia built this. I would however correctly name the package `com.xebia.cakemix`, which is proper form far as I know.

Typesafe, Spring, Twitter, and just about anybody else uses their company name in their packages.